src/main/java/tr/com/srdc/cda2fhir/transform/DataTypesTransformerImpl.java

```diff
@@ -248,7 +248,7 @@ public String tED2Annotation(ED ed, Map<String, String> idedAnnotations) {
 			if (idedAnnotations != null) {
 				// Try to pull the reference.
 				TEL tel = ed.getReference();
-				if (!(tel == null)) {
```

Comment:
isn't this the same thing?

Reply:
Yeah it is the same thing. I just figured it was a little weird doing it this way is all.